tools/export_slow_tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+from tools.stats_utils.s3_stat_parser import get_previous_reports_for_branch, Report, Version2Report
+from typing import cast, Dict, Tuple, List
+
+SLOW_TESTS_FILE = '.pytorch-slow-tests'
+SLOW_TEST_CASE_THRESHOLD = 60.0
```

Comment:
Would it be too verbose to include the word "seconds" in the name here, or perhaps just add a comment to this line saying that it's seconds?

Reply:
```suggestion
SLOW_TEST_CASE_THRESHOLD_SEC = 60.0
```